src/components/productDetails/outfitHelpers.js

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import { fetchProducts } from "../../redux/actions/getProductsHelper.js";
+import { fetchImages } from "../../redux/actions/getImagesHelper.js";
+
+//FUNC TO GRAB OUTFIT IMAGES
+export function fetchOutfitsImages(setOutfitsImages) {
+  let localOutfit = JSON.parse(window.localStorage.getItem("outfit"));
+  if (localOutfit === null) {
+    console.log("no outfits saved in local storage");
+  } else {
+    let OutfitIds = Object.values(localOutfit);
+    fetchImages(OutfitIds)
+      .then(responseArr => {
+        let dataArray = responseArr.map(response => response.data);
+        setOutfitsImages(dataArray);
+      })
+      .catch(err => {
+        console.log("err trying to fetch all related product data: ", err);
```

Comment:
Might be useful to distinguish this error message from the one below

Reply:
Good idea. Will keep in mind.